capstone/src/main/webapp/script.js

```diff
@@ -326,25 +326,23 @@ function isStringEmpty(str) {
   return (str.length === 0 || !str.trim() || !str);
 }
 
-/** post request params to send a POST request using fetch() */
-const requestParamPOST = {
-  method: 'POST',
-  headers: {
-    'Content-Type': 'application/json',
-  },
-};
-
-function getReviews() {
+async function getReviewsEntities(reviews) {
   // TODO(#33): integrate with actual reviews of businesses
-  const review = 'Really good pizza, nice wine, reasonable prices and great music.';
-  getEntities(review);
+  const reviewsEntities = await getEntities(reviews);
+  return reviewsEntities;
 }
 
 /** send POST request to Cloud Natural Language API for entity recognition */
-function getEntities(message) {
-  const url = '/nlp-entity-recognition?message=' + message;
-  fetch(url, requestParamPOST).then((response) => response.json()).then((entities) => {
-    const businessTags = entities;
+function getEntities(messages) {
+  const url = '/nlp-entity-recognition?messages=' + messages;
+  const requestParamPOST = {
```

Comment:
what's the reasoning for moving the requestParamPOST to inside the function? what if in the future, other functions also want to call post requests?

Reply:
I'm not sure why, but when I tested it again I kept getting an error about how I had to initialize `requestParamPOST`, so I moved it to inside

Reply:
bet